src/main/java/com/minecolonies/coremod/blocks/BlockConstructionTape.java

```diff
@@ -209,6 +333,37 @@ public AxisAlignedBB getBoundingBox(final IBlockState state, final IBlockAccess
         }
     }
 
+    /**
+     * @deprecated remove when minecraft invents something better.
+     */
+    @Deprecated
+    @Override
+    public IBlockState getActualState(@NotNull final IBlockState state, @NotNull final IBlockAccess worldIn, @NotNull final BlockPos pos)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1133)